### DISCUSSION TITLE
so-test fails due to docker: error contacting notary server: x509

### DISCUSSION THREAD

**Discussion by emsltek** [Security-Onion-Solutions/securityonion#1975] [Unsupported Versions] [answered]
Version 2.3 standalone installed on VM for eval using NAT / static IP. verified Internet connectivity as ping yahoo.com returns. Am able to access the SOC from another VM on same subnet. so-status shows all green. 

When running sudo so-test, the following error occurs:

`
[bart@seco ~]$ sudo so-test
Replay functionality not enabled!  Enabling Now....

Note that you will need internet access to download the appropriate components
//=========================================================================
Starting tcpreplay...

This could take a while if another Salt job is running. 
Run this command with --force to stop all Salt jobs before proceeding.
//=========================================================================
[ERROR   ] Command 'docker pull --disable-content-trust=false docker.io/securityonion/so-tcpreplay:2.3.2' failed with return code: 1
[ERROR   ] stderr: Error: error contacting notary server: x509: certificate signed by unknown authority
[ERROR   ] retcode: 1
[ERROR   ] {'pid': 45195, 'retcode': 1, 'stdout': '', 'stderr': 'Error: error contacting notary server: x509: certificate signed by unknown authority'}
[ERROR   ] Command 'docker tag securityonion/so-tcpreplay:2.3.2 seco:5000/securityonion/so-tcpreplay:2.3.2' failed with return code: 1
[ERROR   ] stderr: Error response from daemon: No such image: securityonion/so-tcpreplay:2.3.2
[ERROR   ] retcode: 1
[ERROR   ] {'pid': 45217, 'retcode': 1, 'stdout': '', 'stderr': 'Error response from daemon: No such image: securityonion/so-tcpreplay:2.3.2'}
[ERROR   ] Command 'docker push seco:5000/securityonion/so-tcpreplay:2.3.2' failed with return code: 1
[ERROR   ] stdout: The push refers to repository [seco:5000/securityonion/so-tcpreplay]
[ERROR   ] stderr: An image does not exist locally with the tag: seco:5000/securityonion/so-tcpreplay
[ERROR   ] retcode: 1
[ERROR   ] {'pid': 45227, 'retcode': 1, 'stdout': 'The push refers to repository [seco:5000/securityonion/so-tcpreplay]', 'stderr': 'An image does not exist locally with the tag: seco:5000/securityonion/so-tcpreplay'}
[ERROR   ] Failed to pull seco:5000/securityonion/so-tcpreplay:2.3.2: Error 404: manifest for seco:5000/securityonion/so-tcpreplay:2.3.2 not found: manifest unknown: manifest unknown
`

Seems that there's a certificate issue - thoughts?

**Comment by dougburks** [comment, marked as answer]
I just tested `so-test` and it worked for me.

Check and see if you have full outbound Internet access and if perhaps you have a proxy server or firewall that may be intercepting TLS.

**Reply by emsltek** [reply]
I'm getting similar error with curl 

`curl https://yahoo.com
curl: (60) Peer's certificate issuer has been marked as not trusted by the user.
`

Is there some CA bundle that's not being found? or not installed?

**Reply by greatapoc** [reply]
Do you have a firewall or some other device doing SSL decryption?
